pkg/sfu/buffer/rtpstats_base.go

```diff
@@ -474,7 +474,8 @@ func (r *rtpStatsBase) maybeAdjustFirstPacketTime(ets uint64, extStartTS uint64)
 	}
 
 	samplesDuration := time.Duration(float64(samplesDiff) / float64(r.params.ClockRate) * float64(time.Second))
-	now := time.Now()
+	timeSinceFirst := time.Since(r.firstTime)
+	now := r.firstTime.Add(timeSinceFirst)
```

Comment:
Changing this to ensure only monotonic clock is used. Saw some instances where it looked like NTP time might have adjusted and caused anachronous sender reports.